src/reference/supported-platforms.md

```diff
@@ -4,52 +4,78 @@ short-title: Supported platforms
-
-1. **Supported**<br>
-   Google-tested platforms that
-   are automatically tested on every commit
-   by continuous integration testing.
-1. **Best effort**<br>
-   Platforms that we intend to support through
-   coding practices,
-   but are only tested on an ad-hoc basis.
-1. **Unsupported**<br>
-   Platforms that we don't test or support.
-   
-{% comment %}
-**IMPORTANT NOTE**:
-When changing the minimum support version of macOS, Windows, or Linux,
-make sure to make a corresponding change in the get started pages:
-  * /src/get-started/install/macos.md
-  * /src/get-started/install/windows.md
-  * /src/get-started/install/linux.md
-{% endcomment -%}
-
-
-## Deploying Flutter
-
-As of the current stable release,
-support for deploying Flutter apps is shown in the
-following table:
+As of Flutter {{site.appnow.flutter}},
+Flutter supports the following hardware architectures and platform versions
+to run Flutter code.
+
+## Supported hardware architectures
+
+Flutter supports running apps on the following hardware architectures.
+
+{% assign chips = site.data.chipsets %}
+{% capture yes %}
+ <span class="material-symbols" 
+       style="color: #158477"
+       aria-label="The Flutter SDK supports the specified architecture on the specified target platform"
+       role="img">verified</span>
+{% endcapture %}
+{% capture no %}
+ <span class="material-symbols"
+       style="color: #D43324"
+       aria-label="The Flutter SDK does not support the specified
+       architecture on the specified target platform"
+       role="img">dangerous</span>
+{% endcapture %}
+{% capture beta %}
+ <span class="material-symbols"
+       style="color: #13C2AD"
+       aria-label="The Flutter SDK supports ARM64 in beta the only architecture on Windows"
+       role="img">gpp_maybe</span>
+{% endcapture %}
+{% capture na %}
+ <span class="material-symbols"
+       style="color: #DADCE0"
+       aria-label="No version of the Flutter SDK exists for the
+       specified architecture on the specified target platform"
+       role="img">do_not_disturb_on</span>
+{% endcapture %}
+
+| Platform |   x86   |   x64   |  ARM32  |   ARM64   | RV64GC   |
+|---|:---:|:---:|:---:|:---:|:---:|
+{%- for chip in chips %}
+  | {{chip.platform}} | {{chip.x86}} | {{chip.x64}} | {{chip.arm32}} | {{chip.arm64}} | {{chip.risc}} |
+{%- endfor %}
+{:.table.table-striped}
+
+<div aria-hidden="true" markdown="1">
+
+{{yes}} Supported in all channels.  
+{{no}} Unsupported in all channels.  
+{{beta}} Supported in Beta channel only.  
+{{na}} No version exists.  
+
+</div>
+
+## Supported operating systems
+
+Flutter offers three tiers of support for deploying apps to target platforms.
+
+* **Supported**: Google tests these platforms on every commit.
+* **Best effort**: Google intends to support these platforms
+  through coding practices. Google tests these platforms on an ad-hoc basis.
+* **Unsupported**: Google doesn't test or support these platforms.
```

Comment:
I think it's better to not drag "Google" in to all three descriptions. Perhaps it makes sense for the "Supported" category, but the other two are likely better off as "The Flutter team" or "Flutter".

For the **supported** category, I think it also should be "The Flutter team" but can perhaps be strengthened with a mention of Google. I'm not sure exactly how yet, so I'll try to come back to this comment with ideas. Perhaps something like "The Flutter team, supported by Google, runs tests for these platforms on every commit."?

Reply:
That's fair. The original said Google; I was just updating.

Reply:
Fixed.